rsc/extra/check_js_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -eu
+
+# === check_javascript ===
+#
+# This script is intendend to check that alt-ergo-js builds
+# and run correctly with nodejs
+
+# Be sure of where we are, and initialize some variables
+git_repo=`git rev-parse --show-toplevel`
+local_install_dir=$1
+
+# Cd into the root project directory
+cd $git_repo
+
+# Configure using the local installation directory
+./configure --prefix=$local_install_dir
+
+# Check that the makefile targets work
+echo "=+= [check_makefile.sh] building ... =+="
+opam install zarith_stubs_js js_of_ocaml-compiler
```

Comment:
should you install `js_of_ocaml` and `js_of_ocaml-lwt` as well ?

Reply:
Yes some dependencies were missing. This is now manage in the travis script using the dependencies of the js packages